config/apparmor/abstractions/container-base

```diff
@@ -73,6 +73,7 @@
   # block some other dangerous paths
   deny @{PROC}/kcore rwklx,
   deny @{PROC}/sysrq-trigger rwklx,
+  deny @{PROC}/acpi/** rwklx,
```

Comment:
I'm confused as to how the commit message relates to the content.

Reply:
Yeah this is probably out of scope of this PR.
AFAICT this has been added to lxd's generated profile at some point. In fact, there may be more changes in lxd since we added the generated profile to lxc I suppose? But yeah, not as part of this PR.

Reply:
> 

My points is that you're not explaining why you're adding that acpi line in there.

Reply:
> > 
> 
> My points is that you're not explaining why you're adding that acpi line in there.

It's a change that comes from `cat abstractions/container-base.in container-rules > abstractions/container-base` command. This rule was added not by me, but in https://github.com/lxc/lxc/commit/ec90f35b4cfdd9a22e518b7e7801c8fbd55b2f99

Reply:
Aha, I see. I was confused by that.